.ci.yaml

```diff
@@ -2170,7 +2170,10 @@ targets:
 
   - name: Linux_pixel_7pro external_ui_integration_test
     recipe: devicelab/devicelab_drone
-    presubmit: false
+    # This test used to be presubmit: false. It should block engine rolls.
```

Comment:
@yusuf-goog Do we still need to get up to 3 Linux hosts with Pixel 7 attached in the try pool even if this will only run on engine rolls?

Reply:
This is a good point. We have now 2 linux/pixel and 2 mac_x64/ios in the try pool. If we want to enforce the at least 3 for each platform, we need to add capacity for both.

Reply:
To be clear I think we just need an emulator, at least for the Android build, if that helps.

Reply:
Could try adding `contexts`, `device_type`, `dependencies` properties and `dimensions` to the linux pixel target following `android_defines_test`?
```
  - name: Linux_pixel_7pro android_defines_test
    recipe: devicelab/devicelab_drone
    presubmit: true
    timeout: 60
    dimensions:
      kvm: "1"
      cores: "8"
      machine_type: "n1-standard-8"
    properties:
      contexts: >-
        [
          "android_virtual_device"
        ]
      device_type: "none"
      tags: >
        ["devicelab", "linux"]
      task_name: android_defines_test
      dependencies: >-
        [
          {"dependency": "android_virtual_device", "version": "34"}
        ]
```

@ricardoamador seems we are missing the latest `contexts` property in the (WIP) [wiki](https://github.com/flutter/flutter/wiki/How-to-add-a-new-integration-test-to-Framework-CI#how-to-add-an-integration-tests-with-android-emulator-support)

Reply:
Ah yeah I will need to add it. I think this is recent since the version was enough to create it. I will update the wiki.